metron-platform/metron-management/README.md

```diff
@@ -244,10 +244,12 @@ The functions are split roughly into a few sections:
   * Returns: The String representation of the enrichment config
 
 ## Deployment Instructions
-
-Deployment is as simple as dropping the jar created by this project into
-`$METRON_HOME/lib` and starting the Stellar shell via
-`$METRON_HOME/bin/stellar`
+* Ambari
```

Comment:
More detail would be helpful here.  Perhaps "Clusters installed via the Ambari management pack (default)" and "Manual installation"?